syntax = "proto2";

package rpc;

option cc_generic_services = true;

message EchoRequest {
  required string text = 1;
}

message EchoResponse {
  required string text = 1;
}

service EchoService {
  rpc Echo(EchoRequest) returns (EchoResponse);
}
